### PR TITLE
Fixed the unfinished sentence in Javadoc for cucumber.options

### LINKED ISSUE #2690: Unfinished sentence in Javadoc
### 👓 What did you see?

> Thought to point out an unfinished sentence here. It's [still present in `main`](https://github.com/cucumber/cucumber-jvm/blob/main/cucumber-core/src/main/java/io/cucumber/core/options/Constants.java#L124).
> @timtebeek - https://github.com/cucumber/cucumber-jvm/commit/ad47dd5000c4d40151e99d35418ea58b76f41d2c#r98362078

### PR DESCRIPTION
**Pull request by vishesh10**
### 🤔 What's changed?

Fixed the incomplete sentence in Javadoc.

### ⚡️ What's your motivation? 

Fixes #2690

### 🏷️ What kind of change is this?

- :bank: Refactoring (improvement to documentation)

### 📋 Checklist:

- [x] I agree to respect and uphold the [Cucumber Community Code of Conduct](https://cucumber.io/conduct/)
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.

----

### CODE REVIEW

**Comment by timtebeek** [comment]
You'll want to run `mvn spotless:apply` locally to fix the format violations, and then commit the results.

File: cucumber-core/src/main/java/io/cucumber/core/options/Constants.java [hunk 1]
```diff
@@ -130,7 +130,8 @@ public final class Constants {
     /**
      * Property name formerly used to pass command line options to Cucumber:
      * {@value} This property is no longer read by Cucumber. Please use any of
-     * the
+     * the individual properties instead.
+     * See the java doc on io.cucumber.core.options.Constants for details.
```
Comment (mpkorstanje) [inline cucumber-core/src/main/java/io/cucumber/core/options/Constants.java:134]: Pretty good job on reconstructing the original sentence. But this is the Javadoc of io.cucumber.core.options.Constants so telling people to look there for details isn't quite right.

**Comment by vishesh10** [comment]
Apologies for the recent operation. I am trying to figure out how to correct this.

**Comment by mpkorstanje** [comment]
No worries. If you can't reopen, try making a new branch from the old one.

**Pull request closed without merging**